libs/uw-imap/Makefile

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright (C) 2017 Lucian Cristian <lucian.cristian@gmail.com>
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=uw-imap
+PKG_VERSION:=2007f
+PKG_RELEASE:=1
+
+PKG_SOURCE:=imap-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:= \
+	http://ftp.ntua.gr/pub/net/mail/imap \
+	http://gd.tuwien.ac.at/infosys/mail/imap
+
+PKG_HASH:=53e15a2b5c1bc80161d42e9f69792a3fa18332b7b771910131004eb520004a28
+PKG_BUILD_DIR:=$(BUILD_DIR)/imap-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Lucian Cristian <lucian.cristian@gmail.com>
+
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE.txt
+
+PKG_FIXUP:=autoreconf
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/uw-imap
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=University of Washington IMAP toolkit
+  URL:=https://www.washington.edu/imap/
+  DEPENDS:=+libpthread +libopenssl
+endef
+
+define Package/uw-imap/description
+    The UW IMAP tookit includes the following:
+    c-lient library : an API (application programming interface) used to build email clients and servers,
+    including support for IMAP,POP3, SMTP and NNTP protocols and for local mailbox file access on Unix and Windows
+endef
+
+TARGET_CFLAGS += \
+	-DDISABLE_POP_PROXY -fno-strict-aliasing -Wno-pointer-sign -Wno-format \
```

Comment:
NAK on `-Wno-format`. `sprintf()` exploits are a major source of attacks against C programs.  Please fix this.

Also, what happens if you compile with `-Wall`?

Reply:
I just added them so the compile will not complain, for a clean output, there are lots of warnings

Reply:
For me, that's a concern!

We're using old stale code, and for all we know other people have found and fixed many of this issues.

Normand seemed fairly open to accepting patches and doing numbered releases.  Maybe we should go with his fork.